cli/src/main/kotlin/commands/UploadResultToSw360Command.kt

```diff
@@ -172,6 +172,6 @@ class UploadResultToSw360Command : CliktCommand(
 
     private fun getProjectWithPackages(ortResult: OrtResult): Map<Project, List<Package>> =
         ortResult.getProjects(omitExcluded = true).associateWith { project ->
-            project.collectDependencies().mapNotNull { ortResult.getUncuratedPackageById(it) }
+            project.collectDependencies().mapNotNull { ortResult.getPackage(it)?.pkg }
```

Comment:
Interesting finding @mnonnenmacher. However, I believe the original code was correct, considering that we also have `Sw360PackageCurationProvider`. If SW360 itself can be user as a curation provider, it probably makes sense to only unload uncurated packages to it. @bs-ondem, am I correct that this was the intent? If so, that probably deserves a code comment.

Reply:
@sschuberth Yes you're correct. As you already mentioned it, SW360 also serves as a package curation provider, that's why we decided to upload only uncurated packages.

Reply:
Thanks for the explanation, I have created https://github.com/oss-review-toolkit/ort/pull/3641 instead.